test/bundler_smoke_tests/run-jsdom-tests.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
```

Comment:
@timmywil This line, together with `755` permissions, allows to run this file directly, without prepending it with `node `. Does that also work in Windows? If so, this could shorten the npm scripts a bit if we omit the `node` prefix where possible.

Reply:
That does work in Windows. I knew that and left it out, but I don't have a strong preference. Would you rather have that line at the top of the script or use `node` in the npm script? I chose the latter, but I don't mind either way.

Reply:
It's not a big deal & I could go either way but I prefer to have the hashbang line for a few reasons:
1. It clearly states at the top of the file that the file is supposed to be run directly and not just exports some APIs to be used elsewhere
2. 1-2 extra lines in a file don't contribute much to the file length. OTH, the `node ` prefix on long npm scripts does influence readability negatively, especially when repeated for multiple scripts.

Reply:
> It clearly states at the top of the file that the file is supposed to be run directly and not just exports some APIs to be used elsewhere

Many of the modules we use do the latter. I wouldn't want to change those, but I'm fine changing the ones we only use as executables.

> the node  prefix on long npm scripts does influence readability negatively, especially when repeated for multiple scripts.

It does make the npm script longer, but I don't think it's all negative. It helps readability to make it clear it's a JS file running in Node.

Reply:
We discussed this at the meeting. I'll update my file to align with other since it's easier that way; if we ever want to rely on hashbangs, it will be a separate global PR later.

Reply:
Done.